File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P04Ex_FOR_Loop/P05_Salary.java
package P04Ex_FOR_Loop;

import java.util.Scanner;

public class P05_Salary {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        double salary = Double.parseDouble(scanner.nextLine());
        String webSite = "";
        for (; n > 0; n--) {
            webSite = scanner.nextLine();
            switch (webSite) {
                case "Facebook":
                    salary -= 150;
                    break;
                case "Instagram":
                    salary -= 100;
                    break;
                case "Reddit":
                    salary -= 50;
                    break;
            }
        }
        if (salary > 0) {
            System.out.printf("%.0f", salary);
        } else {
            System.out.println("You have lost your salary.");
        }
    }
}

File: JavaScript_Basic_04_2023_SolutionsOfProblems/04.Ex_PB-JS-For-Loop/P05_salary.js
function salary(input) {
    let tabCount = Number(input[0]);
    let salarySum = Number(input[1]);

    for (i = 2; i <= tabCount + 2; i++) {
        tabName = input[i];
        switch (tabName) {
            case "Facebook":
                salarySum -= 150;
                break;
            case "Instagram":
                salarySum -= 100;
                break;
            case "Reddit":
                salarySum -= 50;
                break;
        }
        if (salarySum <= 0) {
            console.log("You have lost your salary.");
            break;
        }
    }
    if (salarySum > 0) {
        console.log(salarySum);
    }
}
salary(["3",
    "500",
    "Facebook",
    "Stackoverflow.com",
    "softuni.bg"])


